manager/general/src/main/java/org/apache/doris/stack/util/CredsUtil.java

```diff
@@ -34,9 +34,9 @@
 public class CredsUtil {
 
     /**
-    * Encrypt Key
-    * AES must be 128 bits
-    */
+     * Encrypt Key
+     * AES must be 128 bits
+     */
     private static final String ENCRYPT_KEY = "12dfA67887iyW321";
```

Comment:
If the private key is directly fixed in the code, it may not work. For the time being, do you want to put it in the configuration file